tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile_integration.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
+
+from pants.util.contextutil import temporary_dir
+from pants_test.backend.jvm.tasks.jvm_compile.base_compile_integration_test import BaseCompileIT
+
+
+class RscCompileIntegration(BaseCompileIT):
```

Comment:
I'd love to see tests of java, scala, and mixed java+scala targets, as there appears to be code which covers each of them?